src/keycloak/chart/values.schema.json

```diff
@@ -154,7 +154,7 @@
       "type": "object",
       "properties": {
         "enabled": {
-          "type": "boolean"
+          "type": ["boolean", "string"]
```

Comment:
Not sure how I feel about this but we need a way to enable this for our CI tests so that we can create a test user, using the admin user. If we make a zarf helm override (as done in this PR) it ends up as a string rather than a boolean ([upstream issue](https://www.github.com/zarf-dev/zarf/issues/2783)). To allow this to template and deploy I adjusted the schema to allow strings, but also adjusted the template to handle strings in a more sane way.